### PR TITLE
Fabo/more participant fixes

### PR DESCRIPTION
**Pull request by faboweb**
More fixes. shows validator images in polkadot and prepares for links to validators

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f6509ba83a2393aab66c06d--lunieio.netlify.app

File: api/lib/reducers/cosmosV0-reducers.js [hunk 1]
```diff
@@ -135,18 +135,18 @@ function tallyReducer(proposal, tally, totalBondedTokens) {
   }
```
Comment (jbibla) [inline api/lib/reducers/cosmosV0-reducers.js:135]: this is cosmosV0 - does this need to be added to V2 as well?

File: api/lib/reducers/cosmosV0-reducers.js [hunk 2]
```diff
@@ -162,7 +162,8 @@ function networkAccountReducer(address, validators) {
   return {
     name: validator ? validator.name : undefined,
     address: address || '',
-    picture: validator ? validator.picture : ''
+    picture: validator ? validator.picture : '',
+    validator
```
Comment (jbibla) [inline api/lib/reducers/cosmosV0-reducers.js:166]: adding the whole validator object as well?

Comment (faboweb) [reply api/lib/reducers/cosmosV0-reducers.js:166]: yes. gql will filter out

File: api/lib/reducers/cosmosV0-reducers.js (hunk 1 above)
Comment (faboweb) [reply api/lib/reducers/cosmosV0-reducers.js:135]: no

**Comment by jbibla** [comment]
missing changelogs

**Review by jbibla — APPROVED** [review]

**Review by jbibla — APPROVED** [review]

File: api/lib/reducers/polkadotV0-reducers.js [hunk 3]
```diff
@@ -516,14 +516,16 @@ function depositReducer(deposit, depositer, network) {
...
+function networkAccountReducer(account, store) {
+  const validator = account ? store.validators[account.accountId] : undefined
   return {
     name:
       account && account.identity && account.identity.display
         ? account.identity.display
         : undefined,
     address: account && account.accountId ? account.accountId : '',
-    picture: account ? account.twitter : '' // TODO: get the twitter picture using scriptRunner
+    picture: validator ? validator.picture : account ? account.twitter : '',
```
Comment (Bitcoinera) [inline api/lib/reducers/polkadotV0-reducers.js:527]: What does this mean? Did you change something to get that Twitter image?

Comment (faboweb) [reply api/lib/reducers/polkadotV0-reducers.js:527]: the validators are already enriched with pictures so we use them from there

Comment (Bitcoinera) [reply api/lib/reducers/polkadotV0-reducers.js:527]: ah ok. And the Polkadot councellors we can get them later

Comment (Bitcoinera) [reply api/lib/reducers/polkadotV0-reducers.js:527]: ah, now I see. Yes, I think I was just sleepy :sweat_smile:

**Pull request merged**